grpc/README.md

```diff
@@ -0,0 +1,34 @@
+# GRPC Example
+
+**Note:** This is an advanced scenario useful for people that want to *manually* instrument their own code.
+
+This is a simple example that demonstrates how to use the OpenTelemetry SDK
+to *manually* instrument a simple GRPC based Client/Server application.
+The example utilize the [opentelemetry-grpc-1.6](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/grpc-1.6/library#library-instrumentation-for-grpc-160)
+to instrument both the GRPC client and server. 
+
+# How to run
+
+## Prerequisites
+* Java 1.8.231
+* Be on the project root folder
```

Comment:
People might think that they need to be in the repo root, not this `grpc` subdir. Can we clarify?